File: DoAnMau/app/src/main/java/com/example/phuongnamlib/Models/Authen.java
package com.example.phuongnamlib.Models;

public class Authen {
    private String role, token;

    public Authen(String role, String token) {
        this.role = role;
        this.token = token;
    }

    public String getRole() {
        return role;
    }

    public void setRole(String role) {
        this.role = role;
    }

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }
}

File: DoAnMau/pnlib/pnlib/Models/Authen.cs
namespace pnlib.Models
{
    public class Authen
    {
        public string role { get; set; }
        public string token { get; set; }
        public Authen(string role, string token)
        {
            this.role = role;
            this.token = token;
        }
    }
}
